<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<ns2:test-suite xmlns:ns2="urn:model.allure.qatools.yandex.ru" start="1417697173075" stop="1417697191732">
    <name>Custom suite</name>
    <test-cases>
        <test-case start="1417697178920" stop="1417697178923" status="canceled">
            <name>userEmptyPassword</name>
            <failure>
                <message>RuntimeException: java.lang.NullPointerException</message>
                <stack-trace>java.lang.RuntimeException: java.lang.NullPointerException
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:143)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:426)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:125)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:134)
Caused by: java.lang.NullPointerException
	at khppp.excel.utils.ExcelReader.getCellData(ExcelReader.java:68)
	at khppp.excel.utils.ExcelReader.getRowData(ExcelReader.java:56)
	at khppp.excel.utils.ExcelReader.getTableArray(ExcelReader.java:44)
	at khppp.core.CoreTest.testData(CoreTest.java:58)
	at khppp.tests.BaseCase.testData(BaseCase.java:48)
	at khppp.tests.LoginTest.credentials(LoginTest.java:26)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:117)
	... 26 more
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="feature" value="Login"/>
            </labels>
        </test-case>
        <test-case start="1417697179014" stop="1417697180697" status="passed">
            <name>userIncorrectLogin[[admin, adni, Incorrect login or password]]</name>
            <steps/>
            <attachments/>
            <labels>
                <label name="feature" value="Login"/>
            </labels>
        </test-case>
        <test-case start="1417697180808" stop="1417697182532" status="passed">
            <name>userLogin[[Admin_Admin, Admin111, Admin Admin (admin)]]</name>
            <steps/>
            <attachments/>
            <labels>
                <label name="feature" value="Login"/>
            </labels>
        </test-case>
        <test-case start="1417697182541" stop="1417697184491" status="passed">
            <name>userLogin[[LabManager_LabManager, LabManager111, LabManager LabManager (lab manager)]]</name>
            <steps/>
            <attachments/>
            <labels>
                <label name="feature" value="Login"/>
            </labels>
        </test-case>
        <test-case start="1417697184494" stop="1417697186046" status="passed">
            <name>userLogin[[Mentor_Mentor, Mentor111, Mentor Mentor (mentor)]]</name>
            <steps/>
            <attachments/>
            <labels>
                <label name="feature" value="Login"/>
            </labels>
        </test-case>
        <test-case start="1417697186049" stop="1417697187656" status="passed">
            <name>userLogin[[Mentee_Mentee, Mentee111, Mentee Mentee (mentee)]]</name>
            <steps/>
            <attachments/>
            <labels>
                <label name="feature" value="Login"/>
            </labels>
        </test-case>
        <test-case start="1417697187660" stop="1417697189741" status="passed">
            <name>userLogin[[LabManager_LabManager, LabManager111, LabManager LabManager (lab manager)]]</name>
            <steps/>
            <attachments/>
            <labels>
                <label name="feature" value="Login"/>
            </labels>
        </test-case>
        <test-case start="1417697189746" stop="1417697191394" status="passed">
            <name>userLogin[[Admin_Admin, Admin111, Admin Admin (admin)]]</name>
            <steps/>
            <attachments/>
            <labels>
                <label name="feature" value="Login"/>
            </labels>
        </test-case>
    </test-cases>
    <labels/>
</ns2:test-suite>
